apps/karma.conf.js

```diff
@@ -47,7 +49,8 @@ module.exports = function (config) {
       plugins: [
         new webpack.ProvidePlugin({React: 'react'}),
         new webpack.DefinePlugin({
-          IN_UNIT_TEST: true,
+          IN_UNIT_TEST: JSON.stringify(true),
```

Comment:
by the way, the lols is because I'm laughing at myself for writing such seemingly strange code.